test/external-e2e/e2etest_storageclass.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   name: testazurelustre.csi.azure.com
 provisioner: azurelustre.csi.azure.com
 parameters:
-  mgs-ip-address: "172.18.16.8"      # dev1, azurelustre cluster for daily work 
-  #mgs-ip-address: "172.18.17.7"      # dev2, azurelustre cluster for long-haul and PR gated check-in
+  mgs-ip-address: "172.18.32.5"
```

Comment:
put a placeholder in this yaml and replace it in start-long-haul.sh, you can find an example in the sh